lti_consumer/templates/html/lti-dl/render_image.html

```diff
@@ -0,0 +1,4 @@
+{% if attrs.title %}<h2>{{ attrs.title }}</h2>{% endif %}
+{% if attrs.text %}<p>{{ attrs.text }}</p>{% endif %}
+
+<img src="{{ attrs.url }}" {% if attrs.width %}width="{{ attrs.width }}"{% endif %} {% if attrs.height %}height="{{ attrs.height }}"{% endif %} />
```

Comment:
Do you think we can add alt property with title or text here?

Reply:
Added title as alt if exists.